.github/workflows/makefile.yml

```diff
@@ -39,6 +49,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
+      with:
+        ref: ${{ inputs.branch }}
 
     - name: Testing GMP(GCC)
-      run: C:\Program` Files\Git\bin\bash.exe .github\workflows\scripts\gmp-gcc-aarch64-test.sh
+      run: |
+        git checkout aarch64-pe-dev -- 
```

Comment:
Do you really want interactive checkout on CI? Wouldn't you want to use the `inputs.branch` parameter here?

Reply:
it should be always default branch with workflow scripts. However thanks for pointing out, it was missing folder name for workflows